.gitignore

```diff
@@ -120,3 +122,7 @@ ENV/
 
 # emacs buffers
 \#*
+
+#deps
+/.deps
+
```

Comment:
don't change .gitignore with changes that are not relevant to the topic of the PR: none of these changes seem relevant to the point of the PR.

Reply:
I have removed any changes in this file. The reason I added them was because of the following files getting created on every test execution
![image](https://github.com/Parsl/parsl/assets/22280243/b97c9af4-d4d8-4500-b423-f1b3e0aaa042)

Let me open another PR for this!

Reply:
ok - you can also fix the MPI test perhaps. It looks like: parsl/tests/test_mpi_apps/test_mpi_mode_enabled.py uses `local_setup` and `local_teardown` when it could use `local_config` instead to supply the test configuration. If the test used `local_config`, then the test system should make those files appear under `.pytest/parsltest-current` like it does for many other tests.